### PR TITLE
Prince/ auto login on bot from derivp app

### PR DESCRIPTION
**Pull request by prince-deriv**
## Changes

-   ...

## Type of change

-   [ ] Bug fix
-   [ ] New feature
-   [ ] Update feature
-   [ ] Refactor code
-   [ ] Translation to code
-   [ ] Translation to crowdin
-   [ ] Script configuration
-   [ ] Improve performance
-   [ ] Style only
-   [ ] Dependency update
-   [ ] Documentation update
-   [ ] Release

### CODE REVIEW

File: templates/deriv/localstorage-sync.mustache [hunk 1]
```diff
@@ -2,28 +2,56 @@
...
+                        // if token params aren't captured, force reload the parent window.
+                        parent.location.reload();
+                    }
+
+                    const { origin } = window.location;
+
+                    const login_url = origin + `?${token_params}`;
+
+                    window.location.href = login_url;
+                    console.log(login_url)
```
Comment (matin-deriv) [inline templates/deriv/localstorage-sync.mustache:44]: Is this necessary?

File: templates/deriv/localstorage-sync.mustache [hunk 2]
```diff
@@ -2,28 +2,56 @@
...
                 if (allowed_origins.includes(message.origin)) {
                     switch (message.data.key) {
-                        case 'active_loginid': 
-                        case 'client.accounts': {
-                            if (localStorage.getItem(message.data.key) !== message.data.value) {
-                                localStorage.setItem(message.data.key, message.data.value);
+                        case 'active_loginid':
+                        case 'client.accounts':
+                            {
+                                autoLogin();
```
Comment (balakrishna-deriv) [inline templates/deriv/localstorage-sync.mustache:52]: how does it log out?

Comment (prince-deriv) [reply templates/deriv/localstorage-sync.mustache:52]: For logout it's in the iframe linked to deriv-app. Whatever is the token we have in binary-bot will get send to deriv-app no changes required for that iframe and incase we need, the changes should be on the file inside deriv app.

File: templates/deriv/localstorage-sync.mustache (hunk 1 above)
Comment (prince-deriv) [reply templates/deriv/localstorage-sync.mustache:44]: console here can be removed. The rest are necessary to mimic the real  login process of the user.

**Review by balakrishna-deriv — APPROVED** [review]

File: templates/deriv/localstorage-sync.mustache [hunk 3]
```diff
@@ -2,28 +2,57 @@
...
-                    'https://app.deriv.com', 
-                    'https://app.deriv.be', 
-                    'https://app.deriv.me', 
+                    'https://app.deriv.com',
+                    'https://app.deriv.be',
+                    'https://app.deriv.me',
                     'https://staging-app.deriv.com',
                     'https://staging-app.deriv.be',
                     'https://staging-app.deriv.me',
+                    'https://localhost.binary.sx',
```
Comment (balakrishna-deriv) [inline templates/deriv/localstorage-sync.mustache:16]: do we need this?

Comment (balakrishna-deriv) [reply templates/deriv/localstorage-sync.mustache:16]: maybe we can comment it out

**Review by balakrishna-deriv — APPROVED** [review]

**Pull request merged**